src/main/resources/component.mustache

```diff
@@ -1,12 +1,17 @@
 The Snowdrop team is pleased to inform you that we're planning to release a new Snowdrop BOM on *{{releaseDate}}*
 based on Spring Boot *{{version}}*.
 
-Some of our artifacts depend on your product and we need to know which product version we should be using for this new
+Some of our artifacts depend on your component and we need to know which component version we should be using for this new
 release.
 
-While we don't require a new release of your product as long as the artifacts you provide us with are tested and supported
-on Spring Boot *{{version}}*, it's important that you let us know until when your product is supported so that we can plan
-future releases accordingly. For reference, we will support this new release until *{{endOfSupportDate}}*.
+{{#starter}}
+We don't require a new release of your component as long as the artifacts you provide us with are tested and supported
+on Spring Boot *{{version}}*.
+{{/starter}}
+{{#release}}
+It's important that you let us know until when your product is supported so that we can plan future releases accordingly.
```

Comment:
I suggest to use `It is important that you also communicate the EOL of your product so that we can plan our future releases accordingly.`

Reply:
Done.